src/components/Welcome.jsx

```diff
@@ -1,128 +1,132 @@
-import React, { useContext } from "react";
-import { AiFillPlayCircle } from "react-icons/ai";
-import { SiEthereum } from "react-icons/si";
-import { BsInfoCircle } from "react-icons/bs";
+import React, { useContext } from 'react'
+import { AiFillPlayCircle } from 'react-icons/ai'
+import { SiEthereum } from 'react-icons/si'
+import { BsInfoCircle } from 'react-icons/bs'
 
-import { TransactionContext } from "../context/TransactionContext";
-import { shortenAddress } from "../utils/shortenAddress";
-import { Loader } from ".";
+import { transactionContext } from '../context/TransactionContext'
+import { shortenAddress } from '../utils/shortenAddress'
+import { Loader } from '.'
 
-const companyCommonStyles = "min-h-[70px] sm:px-0 px-2 sm:min-w-[120px] flex justify-center items-center border-[0.5px] border-gray-400 text-sm font-light text-white";
+const companyCommonStyles =
+  'min-h-[70px] sm:px-0 px-2 sm:min-w-[120px] flex justify-center items-center border-[0.5px] border-gray-400 text-sm font-light text-white'
 
 const Input = ({ placeholder, name, type, value, handleChange }) => (
-    <input
-        placeholder={placeholder}
-        type={type}
-        step="0.0001"
-        value={value}
-        onChange={(e) => handleChange(e, name)}
-        className="my-2 w-full rounded-sm p-2 outline-none bg-transparent text-white border-none text-sm white-glassmorphism"
-    />
-);
+  <input
+    placeholder={placeholder}
+    type={type}
+    step="0.0001"
+    value={value}
+    onChange={(e) => handleChange(e, name)}
+    className="my-2 w-full rounded-sm p-2 outline-none bg-transparent text-white border-none text-sm white-glassmorphism"
```

Comment:
It runs well on dev server and can build normally for me, all functions we had before could work, including connecting the wallet, donateForFood, getAllPuppies and getAllTransactions.

Every pages look well. But there's some difference in the Welcome page. I think the input part looks prettier without the "white-glassmorphism" style. We have added this style before too, but some how it didn't work on my page before, and I think it's prettier without it. I suggest to remove it but it's also fine to keep it.

```suggestion
    className="my-2 w-full rounded-sm p-2 outline-none bg-transparent text-white border-none text-sm"
```

Reply:
Nice advice
addressed in [3c075b9](https://github.com/lohsuan/puppy_sponsor/pull/5/commits/3c075b93627ffbfe611237617f100ad6705e21c2).